src/js/index.js

```diff
@@ -0,0 +1,164 @@
+import { userInput, weatherCard } from './content';
+
+const root = document.getElementById('root');
+const body = document.querySelector('body');
+
+body.classList.add('bg-center', 'bg-cover', 'bg-no-repeat');
+root.classList.add('h-screen', 'w-full', 'opacity-95');
+body.style.backgroundImage = 'url("images/main-background.jpg")';
+
+root.appendChild(userInput());
+weatherCard();
+
+// user input section
+
+const form = document.querySelector('form');
+const cityInput = document.querySelector('input');
+const cel = document.querySelector('.celcius');
+const farh = document.querySelector('.farhenheit');
+
+const modal = document.getElementById('myModal');
+
+// weather fields
+
+const cardTime = document.querySelector('.card-time');
+const cardCity = document.querySelector('.card-city');
+const cardTemp = document.querySelector('.card-temp');
+const cardHumi = document.querySelector('.humidity');
+const cardPress = document.querySelector('.pressure');
+const cardSpeed = document.querySelector('.speed');
+const cardDesc = document.querySelector('.description');
+
+let city;
+let scale = false;
+let show;
+const cScale = ' &#176;C';
+const fScale = ' &#176;F';
+let temperature;
+
+// modal
+
+const modalWarn = () => {
+  modal.style.display = 'block';
+
+  window.onclick = function (event) {
+    if (event.target === modal) {
+      modal.style.display = 'none';
+    }
+  };
+};
+
+// Get weather data
+
+async function getData(city) {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=49257f6591cfc3ed8daf0b5970d519cb`,
+    { mode: 'cors' });
+
+  const data = await response.json();
```

Comment:
- [ ] Please change this to es6 arrow function syntax. Do this for other regular functions in the code too. It's important to use es6+ rules.